packages/mobile/scripts/translateFile.js

```diff
@@ -7,7 +7,7 @@ const fileName = process.argv[2]
 const googleApiToken = process.argv[3]
 console.info(`Translating file: ${fileName}`)
 
-const json = fs.readFileSync(`../locales/en-US/${fileName}`)
+const json = fs.readFileSync(`../locales/base/${fileName}`)
```

Comment:
i wasn't sure this guy and verify_locales were still being used, but i updated them just in case

Reply:
Ah yes, this script should be deleted in another PR.